Doc/Tutorial/chapter_pdb.rst

```diff
@@ -62,6 +62,23 @@ Example: get the list of the :math:`y` coordinates of all atoms
 
    >>> y_list = mmcif_dict["_atom_site.Cartn_y"]
 
+Reading a BinaryCIF file
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Create a ``BinaryCIFParser`` object:
+
+.. code:: pycon
+
+   >>> from Bio.PDB.binary_cif import BinaryCIFParser
+   >>> parser = BinaryCIFParser()
+
+Call ``get_structure`` with the path to the BinaryCIF file:
+
+.. code:: pycon
+
+   >>> structure = parser.get_structure("1GBT.bcif")
+
```

Comment:
Those really ought to be tested examples (the same is true for many of the examples nearby in the tutorial, like the MMTF one). Try adding this line above the ``.. code:: pycon`` entry:

```

.. doctest ../Tests/PDB lib:numpy lib:msgpass

```

See https://github.com/biopython/biopython/blob/master/Doc/Tutorial/chapter_testing.rst and https://github.com/biopython/biopython/blob/master/Tests/test_Tutorial.py